docs/databricks-monitoring/databricks-monitoring.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Monitoring Azure Databricks in Azure Log Analytics
+titleSuffix: A library for monitoring Azure Databricks in Azure Monitor
+description: A scala library to enable monitoring of metrics and logging data in Azure Log Analytics
+author: petertaylor9999
+ms.date: 02/28/2019
+ms.topic:
+ms.service:
+ms.subservice:
+---
+
+# Monitoring Azure Databricks in Azure Log Analytics
+
+[Azure Databricks](/azure/azure-databricks/) is a fast, powerful, and collaborative [Apache Spark](https://spark.apache.org/)–based analytics service that makes it easy to rapidly develop and deploy big data analytics and artificial intelligence (AI) solutions. Most users take advantage of the simplicity of notebooks in their Azure Databricks solutions. For users that require more robust computing options, Azure Databricks supports the distributed execution of custom application code written in Scala.
```

Comment:
Notebooks can use Scala, Python, SQL, and R.  Non-notebook applications can be written in Python or compiled from Java/Scala.

Reply:
there is a small typo in https://github.com/mspnp/architecture-center/blob/peter/databricks/docs/databricks-monitoring/databricks-performance-dashboard.md
below bold

Sum Task Execution per host
This visualization shows the sum of task execution latency per host running on a cluster. Viewing task execution latency per host identifies hosts that have much higher overall task latency **thatn** other hosts. This may mean that tasks have been inefficiently or unevenly distributed to hosts.